handlers/messageCreate.go

```diff
@@ -80,4 +80,6 @@ func MessageCreateHandler(s *discordgo.Session, m *discordgo.MessageCreate) {
 		}
 		return
 	}
+
+	HandleXpGive(s, m)
```

Comment:
Make this a go routine (add to to the front of it) since it's not returning anything and won't block the overall function itself